boards/arm/b_u585i_iot02a/doc/index.rst

```diff
@@ -195,7 +195,7 @@ The Zephyr b_u585i_iot02a board configuration supports the following hardware fe
 +-----------+------------+-------------------------------------+
 | AES       | on-chip    | crypto                              |
 +-----------+------------+-------------------------------------+
-| RADIO     | on-chip    | Bluetooth Low Energy (BLE)          |
+| RADIO     | STM32WB5MMG| Bluetooth Low Energy (BLE)          |
```

Comment:
Please do any changes in the faulty commits directly. We don't merge fix commits